### PR TITLE
Cost Factors for Basic Set Equipment Modifiers

### PR DESCRIPTION
**Pull request by dec-00**
Added versions of the Basic Set Equipment modifiers that use the Cost Factor system introduced in Low-Tech. I copied over the Note from High-Tech Equipment Modifiers because it's a very good explanation of why to use Cost Factor versions.

### CODE REVIEW

File: Library/Basic Set/Basic Set Equipment Modifiers.eqm [hunk 1]
```diff
@@ -3,231 +3,480 @@
...
+					"name": "Weapon Quality"
+				},
+				{
+					"id": "0bbd759b-51c1-4431-a410-46cb09b5b9f7",
+					"type": "eqp_modifier",
+					"name": "Silver",
+					"reference": "B275",
+					"notes": "+2 to break",
+					"cost_type": "to_base_cost",
+					"cost": "+18 CF"
```
Comment (Calmquist) [inline Library/Basic Set/Basic Set Equipment Modifiers.eqm:224]: Shouldn't this be +19?

File: Library/Basic Set/Basic Set Equipment Modifiers.eqm [hunk 2]
```diff
@@ -3,231 +3,480 @@
...
+				},
+				{
+					"id": "c095f89e-cc51-4203-83c7-e630eb77f4d5",
+					"type": "eqp_modifier",
+					"name": "Silver-coated",
+					"reference": "B275",
+					"notes": "Reduced wounding multiplier to vulnerable targets: x2 becomes x1.5, x3 becomes x2, x4 becomes x3",
+					"cost_type": "to_base_cost",
+					"cost": "+1 CF"
```
Comment (Calmquist) [inline Library/Basic Set/Basic Set Equipment Modifiers.eqm:233]: Shouldn't this be +2?

File: Library/Basic Set/Basic Set Equipment Modifiers.eqm [hunk 3]
```diff
@@ -3,231 +3,480 @@
...
+				},
+				{
+					"id": "1ec94aaf-2041-49e5-b33c-b7741a73d8d8",
+					"type": "eqp_modifier",
+					"name": "Silver Bullets",
+					"reference": "B275",
+					"cost_type": "to_base_cost",
+					"cost": "+48 CF"
```
Comment (Calmquist) [inline Library/Basic Set/Basic Set Equipment Modifiers.eqm:241]: Shouldn't this be +49?

File: Library/Basic Set/Basic Set Equipment Modifiers.eqm [hunk 4]
```diff
@@ -3,231 +3,480 @@
...
+					"name": "Weapon Quality"
+				},
+				{
+					"id": "14b76b22-60e4-42ff-94c2-d3ae2d0fb1b9",
+					"type": "eqp_modifier",
+					"name": "Silver",
+					"reference": "B275",
+					"notes": "+2 to break",
+					"cost_type": "to_base_cost",
+					"cost": "x19"
```
Comment (Calmquist) [inline Library/Basic Set/Basic Set Equipment Modifiers.eqm:459]: Shouldn't this be x20?

File: Library/Basic Set/Basic Set Equipment Modifiers.eqm [hunk 5]
```diff
@@ -3,231 +3,480 @@
...
+				},
+				{
+					"id": "d94a64f0-47ba-48e6-9cb1-285dd5d0c780",
+					"type": "eqp_modifier",
+					"name": "Silver-coated",
+					"reference": "B275",
+					"notes": "Reduced wounding multiplier to vulnerable targets: x2 becomes x1.5, x3 becomes x2, x4 becomes x3",
+					"cost_type": "to_base_cost",
+					"cost": "x2"
```
Comment (Calmquist) [inline Library/Basic Set/Basic Set Equipment Modifiers.eqm:468]: Shouldn't this be x3?

File: Library/Basic Set/Basic Set Equipment Modifiers.eqm [hunk 6]
```diff
@@ -3,231 +3,480 @@
...
+				},
+				{
+					"id": "c49a7edd-1c86-4e0f-962b-c4e1b083060c",
+					"type": "eqp_modifier",
+					"name": "Silver Bullets",
+					"reference": "B275",
+					"cost_type": "to_base_cost",
+					"cost": "x49"
```
Comment (Calmquist) [inline Library/Basic Set/Basic Set Equipment Modifiers.eqm:476]: Shouldn't this be x50?

**Review by Calmquist — CHANGES_REQUESTED** [review]
Unless the silver modifiers have changed since my edition, they are off by one.

**Comment by Calmquist** [comment]
I do like adding cost factors.

**Comment by dec-00** [comment]
Ah yes. My bad. In my haste I read them as multiply by x instead of add x to the price. I'll fix that.

**Review by Calmquist — APPROVED** [review]

**Pull request merged**